prime-router/src/main/kotlin/azure/Authentication.kt

```diff
@@ -1,121 +0,0 @@
-package gov.cdc.prime.router.azure
```

Comment:
I combined Authentication.kt and OktaAuthentication.kt, because they were confusingly redundant.     So all that functionality is now in OktaAuthentication.kt.